tests/test_dockerflow.py

```diff
@@ -0,0 +1,51 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import json
+import os
+import binascii
+
+import click
+import pytest
+
+from landoui.app import create_app
+
+
+@pytest.fixture
+def app():
+    """Needed for pytest-flask."""
+    return click.Context.invoke(
+        None,
+        create_app,
+        run_dev_server=False,
+        debug=True,
+        port=80,
+        host='0.0.0.0',
+        version_path='/version.json',
+        secret_key=str(binascii.b2a_hex(os.urandom(15))),
+        session_cookie_name='landoui-test',
+        session_cookie_domain='',
+        session_cookie_secure=False,
+    )
+
+
+def test_dockerflow_lb_endpoint_returns_200(client):
+    assert client.get('/__lbheartbeat__').status_code == 200
+
+
+def test_dockerflow_heartbeat_endpoint_returns_200(client):
+    assert client.get('/__heartbeat__').status_code == 200
+
+
+def test_dockerflow_version_endpoint_response(client):
+    response = client.get('/__version__')
+    assert response.status_code == 200
+    assert response.content_type == 'application/json'
+
+
+def test_dockerflow_version_matches_disk_contents(client):
+    with open('/version.json') as version_file:
```

Comment:
It would be better to use my `versionfile` fixture in the `lando-api/tests/test_app.py` file.  It avoids hard-coding the file path so the tests will run in any environment.